modules/admin_manual/pages/maintenance/migrating.adoc

```diff
@@ -92,12 +92,26 @@ long as you have access to the physical server you can always log in. In
 the event that a load-balancer is in place, there will be no issues as
 long as it sends the correct `X-Forwarded-Host` header.
 
+In case of a docker based setup, the trusted_domains setting is controlled by the environment variables `OWNCLOUD_TRUSTED_DOMAINS` or `OWNCLOUD_DOMAIN`. The latter takes only effect, if `OWNCLOUD_TRUSTED_DOMAINS` is undefined and can provide one IP-addres or hostname. `OWNCLOUD_TRUSTED_DOMAINS` can specify multiple values as a comma-separated list. 
+
+Here is an example, how this can be used from within a docker-compose.yml file to allow access to ownCloud under two differnet names, and one IP-address, in addition to localhost and 127.0.0.1:
+
+[source,yaml,subs="attributes+"]
+----
+services:
+  owncloud:
+    image: "owncloud/server:{latest-server-version}"
+    environment:
+      OWNCLOUD_TRUSTED_DOMAINS: "myowncloud.mydomain.com, myowncloud, 12.23.34.45"
```

Comment:
Does it work OK with a space character after each comma?

Reply:
yes :-) it does:
```
  if (getenv('OWNCLOUD_TRUSTED_DOMAINS') != '') {
    $domain = array_map('trim', explode(',', getenv('OWNCLOUD_TRUSTED_DOMAINS')));
  } else {
    $domain = [explode(':', getenv('OWNCLOUD_DOMAIN'))[0]];
  }
```